doc/specifications/rest/REST-API-V2.rst

```diff
@@ -6537,6 +6537,164 @@ Example
 
 
 
+Permissions
+===========
+
+
+Overview
+--------
+
+Roles and policies define specific forms of access to resources, and can be combined to achieve complex results.
+When interpreted they result in the permissions set for each user/resource combination.
+
+================================================================= =================== ======================= ============================ ================ ==============
+Resource                                                          POST                GET                      PATCH/PUT                   DELETE           COPY
+----------------------------------------------------------------- ------------------- ----------------------- ---------------------------- ---------------- --------------
+/permissions/content/objects/<ID>                                 .                   load permissions for a  .                            .                .
+                                                                                      content
+/permissions/content/locations/<path>                             .                   load permissions for a  .                            .                .
+                                                                                      location
+================================================================= =================== ======================= ============================ ================ ==============
+
+
+Specification
+-------------
+
+Managing Permissions
+~~~~~~~~~~~~~~~~~~~~
+
+
+Load Permissions for a Content
+``````````````````````````````
+:Resource: /permissions/content/objects/<ID>
+:Method: GET
+:Description: Loads the permissions for the content object with the given id, for the current user. Depending on the Accept header the content object is embedded.
```

Comment:
yes, embedding should not be mentioned / tackled here.

But there should be links both ways, from content to permissions for the content and from permission back to content it represents permission for.